packages/components/addon/components/hds/table/index.hbs

```diff
@@ -1,4 +1,7 @@
 <table class={{this.classNames}} data-test-table ...attributes>
+  {{#if this.isSortableTable}}
+    <caption class="sr-only" aria-live="polite">{{this.sortedMessageText}}</caption>
```

Comment:
I like how we moved this message to be a caption 🙌

Reply:
Thanks @alex-ju! After you pointed out that the generated ID was not needed, it occurred to me that I needed to better associate the message with the table just in case there were multiple sortable tables on the same page, and using the `caption` seemed to be an acceptable way to do that.